### PR TITLE
SA-2247: Remove most of inbox placement pages

### PR DESCRIPTION
**Pull request by PatrickSison**
### What Changed

- Removed inbox placement pages
- Separated seedlist page
- Moved seedlist page to `Configuration` top level nav

### How To Test

- Go to an account
- Verify inbox placement nav item no longer exists
- Verify that the Seedlist page is under the configuration nav item
- Verify that the seedlist page still is under the inbox placement feature flag

### To Do

- [ ] Address feedback

### CODE REVIEW

**Comment by shrirupa** [comment]
Screen Shot 2021-03-10 at 3 45 17 PM
does this description need to change too ?

File: src/config/routes/inboxPlacement.js [hunk 1]
```diff
@@ -8,35 +8,8 @@ export default [
...
-  {
-    path: '/inbox-placement',
-    component: inboxPlacement.TestListPage,
-    layout: App,
-    condition: hasGrants('inbox-placement/manage'),
-    title: 'Inbox Placement Tests',
-    supportDocSearch: 'inbox placement',
-    category: 'Inbox Placement',
+    title: 'Seedlist',
+    supportDocSearch: 'seedlis4t',
```
Comment (shrirupa) [inline src/config/routes/inboxPlacement.js:12]: typo may be 'seedlist' ?

File: src/pages/inboxPlacement/SeedListPage.js [hunk 2]
```diff
@@ -127,18 +127,7 @@ export const SeedListPage = props => {
...
-        as: PageLink,
-        content: 'All Tests',
-        to: '/inbox-placement',
-      }}
-      title={isHibanaEnabled ? 'Inbox Placement Data' : 'Create an Inbox Placement Test'}
-    >
-      {isHibanaEnabled ? renderHibana() : renderOG()}
-    </Page>
-  );
+  return <Page title="Seedlist">{isHibanaEnabled ? renderHibana() : renderOG()}</Page>;
```
Comment (shrirupa) [inline src/pages/inboxPlacement/SeedListPage.js:130]: upto you but we can remove the isHibanaEnabled check now 🙂

File: src/selectors/inboxPlacement.js [hunk 3]
```diff
@@ -1,92 +1,8 @@
...
-  filters,
-});
-
-const getFilters = (state, props) => props.match.params;
-const getInboxPlacementByMailbox = state => state.inboxPlacement.placementsByProvider;
-const getInboxPlacementByRegion = state => state.inboxPlacement.placementsByRegion;
-const getInboxPlacementBySendingIp = state => state.inboxPlacement.placementsBySendingIp;
 
 export const selectReferenceSeed = createSelector([getSeeds], seeds =>
   seeds.find(seed => endsWith(seed, 'seed.sparkpost.com')),
```
Comment (shrirupa) [inline src/selectors/inboxPlacement.js:7]: not sure if we can remediate this Codeql warning 🤷  may be just ignore it.

**Review by shrirupa — COMMENTED** [review]
just had a few comments, overall looks good!

File: src/selectors/inboxPlacement.js (hunk 3 above)
Comment (shrirupa) [reply src/selectors/inboxPlacement.js:7]: may be moving this to constants file can help

File: src/config/routes/inboxPlacement.js (hunk 1 above)
Comment (PatrickSison) [reply src/config/routes/inboxPlacement.js:12]: Oops typo

**Review by shrirupa — CHANGES_REQUESTED** [review]
Screen Shot 2021-03-12 at 2 29 50 PM
Sorry didn't notice this earlier, but that bottom subnav disappears when seedlist is clicked

File: src/config/routes/signals.js [hunk 4]
```diff
@@ -257,6 +257,7 @@ const signalsRoutes = [
     title: 'Signals Integration',
     supportDocSearch: 'signals',
     category: 'Configuration',
+    subcategory: 'Signals Integration',
```
Comment (shrirupa) [inline src/config/routes/signals.js:260]: 🎉

**Review by shrirupa — APPROVED** [review]
Looks good to me!

**Pull request merged**